packages/mobile/package.json

```diff
@@ -7,7 +7,7 @@
     "android": "ENVFILE=.env.prod react-native run-android --variant=prodDebug",
     "android:stage": "ENVFILE=.env.stage react-native run-android --variant=bounceDebug --appIdSuffix bounce",
     "android:dev": "ENVFILE=.env.dev react-native run-android --variant=prodDebug",
-    "ios": "react-native run-ios",
+    "ios": "react-native run-ios --simulator \"iPhone 14 Pro\"",
```

Comment:
Do we want this to be hardcoded? or should this be something configured in the dev's machine as the default instead?

Reply:
React Native seems to default to 13 (at least on this version), which isn't included in the latest XCode by default – I think this is a safe approach